fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateRDFSourcePersister.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.ocfl.impl;
+
+import org.fcrepo.kernel.api.operations.NonRdfSourceOperation;
+import org.fcrepo.persistence.ocfl.api.OCFLObjectSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.fcrepo.kernel.api.operations.ResourceOperationType.CREATE;
+
+/**
+ * This class implements the persistence of a new RDFSource
+ *
+ * @author dbernstein
+ * @since 6.0.0
+ */
+public class CreateRDFSourcePersister extends AbstractPersister<NonRdfSourceOperation> {
+
+    private static Logger LOGGER = LoggerFactory.getLogger(CreateRDFSourcePersister.class);
```

Comment:
most other logger declarations in the project are `private static final Logger log`, it'd be good to use a consistent name

Reply:
For some reason this disappeared from the main list of comments ^

Reply:
done.